src/bin/miri.rs

```diff
@@ -318,6 +319,19 @@ fn main() {
                     };
                     miri_config.cmpxchg_weak_failure_rate = rate;
                 }
+                arg if arg.starts_with("-Zmiri-isolated-op=") => {
+                    let action = match arg.strip_prefix("-Zmiri-isolated-op=").unwrap() {
+                        "hide" => miri::IsolatedOp::Reject(miri::RejectOpWith::NoWarning),
```

Comment:
I am not sure I like `hide` as name for "reject the operation, return error code, but do not print a warning"... but I don't have a better idea either. @oli-obk @bjorn3 @hyd-dev any suggestions?

Reply:
Maybe return-error-code or something like that?

Reply:
Hm, but `warn`/`warn-nobacktrace` also return an error code so they should then have a similar name.

I am also wondering about the name of the flag itself... does "isolated" operation make sense? With "allow", it is not isolated any more, after all. Maybe "external" operation would be a better name? Any other suggestions?

Reply:
I think merging the isolation switch setting and isolation error visibility setting into a single flag is causing the confusion. One more option is to use two flags, one to control isolation switch -- `-Zmiri-disable-isolation` (which already exists); one to control isolation error visibility -- `-Zmiri-isolation-error={ignore|warn|warn-nobacktrace}`.

Reply:
That idea is appealing, but then we have nonsensical combinations such as `-Zmiri-disable-isolation -Zmiri-isolation-error=ignore`.

Reply:
It shouldn't be too hard to catch such combinations in miri's cmd line parser and throw error about it.

Reply:
I updated the PR with a new flag `-Zmiri-isolation-error`.